server/src/main/java/org/opensearch/action/admin/cluster/decommission/awareness/put/DecommissionRequest.java

```diff
@@ -28,23 +29,36 @@
  */
 public class DecommissionRequest extends ClusterManagerNodeRequest<DecommissionRequest> {
 
+    public static final TimeValue DEFAULT_NODE_DRAINING_TIMEOUT = TimeValue.timeValueSeconds(300);
```

Comment:
Let's keep default to 2 min? As the node was already weighed away and 5 min can be too large a number just for inflight requests?

Reply:
Done.